src/test/java/com/google/sps/data/AvailabilityTimeSlotGeneratorTest.java

```diff
@@ -232,7 +232,40 @@ public void tooLargeNegativeOffset() {
     Assertions.assertThrows(
         IllegalArgumentException.class,
         () -> {
-          AvailabilityTimeSlotGenerator.timeSlotsForDay(Instant.now(), -740);
+          AvailabilityTimeSlotGenerator.timeSlotsForWeek(Instant.now(), -740);
         });
   }
+
+  @Test
+  public void createAWeekOfTimeSlots() {
```

Comment:
This might require you to write out the expected for the entire week and not just for the first of each day but l know that's a pain. We can see what @acmcarther and @jalexanderqed have to say.

Reply:
Yep. Jack suggested just the first slots so that it checks the dates and order of the days at least.